### PR TITLE
 Add pytest option --topology-szn-dir

### PR DESCRIPTION
**Pull request by jpmh1309**
This new option will allow to pass the path of a directory
where topologies files *.szn will be defined.

### CODE REVIEW

**Comment by jpmh1309** [comment]
Feedback applied!! @carlos-jenkins @dajose PR back to you!

File: lib/topology/pytest/plugin.py [hunk 1]
```diff
@@ -267,6 +270,11 @@ def pytest_addoption(parser):
         default=None,
         help='Path to a directory where logs are to be stored'
     )
+    group.addoption(
+        '--topology-base-szn-dir',
+        default=None,
```
[the remark this answers is not in the document]
Comment (dajose) [reply lib/topology/pytest/plugin.py:276]: what happens when this guy with append is not used at all? does it returns a empty list when you do the config.getoption?

File: lib/topology/pytest/plugin.py [hunk 2]
```diff
@@ -67,14 +67,17 @@ class TopologyPlugin(object):
     :param dict injected_attr: A dictionary holding topology attributes to
      inject.
     :param str log_dir: Path where to store logs.
+    :param list base_szn_dir: List of paths to directories where *.szn files
```
[the remark this answers is not in the document]
Comment (jpmh1309) [reply lib/topology/pytest/plugin.py:70]: It sounds good :+1:, I'll do it!

**Comment by jpmh1309** [comment]
Feedback applied!! @carlos-jenkins @dajose PR back to you!

**Review by dajose — APPROVED** [review]

**Pull request merged**